packages/application-generic/src/usecases/compile-template/compile-template.usecase.ts

```diff
@@ -50,6 +50,21 @@ Handlebars.registerHelper(
   }
 );
 
+Handlebars.registerHelper(
+  HandlebarHelpersEnum.UNIQUE,
+  function (array, property) {
+    if (!Array.isArray(array)) return '';
+
+    return array
+      .map((item) => {
+        if (item[property]) {
+          return item[property];
+        }
+      })
+      .filter((value, index, self) => self.indexOf(value) === index);
```

Comment:
If I am getting it right, we will return an array of only the property we filtered the distinct of. I feel we should return the whole object. 
For example:
`array: [{name:1,var:1},{name:1,var:1},{name:2,var:1}]` 
should return `
`[{name:1,var:1},{name:2,var:1}]`
instead of ['1','2']

what do you think?